app/src/main/res/layout/customise_card_dialog.xml

```diff
@@ -0,0 +1,96 @@
+            tools:layout_editor_absoluteX="10dp" />
+
+        <ScrollView
+            android:id="@+id/customiseCardScrollView"
+            android:layout_width="398dp"
+            android:layout_height="160dp"
+            android:layout_marginTop="@dimen/input_spacing_large"
+            app:layout_constraintBottom_toTopOf="@id/closeButton"
+            app:layout_constraintTop_toBottomOf="@id/customiseCardTitleTextView"
+            tools:layout_editor_absoluteX="-1dp">
+
+            <LinearLayout
+                android:id="@+id/customiseCardLayout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <ImageView
+                    android:layout_width="match_parent"
+                    android:layout_height="1dp"
+                    android:alpha="0.4"
+                    android:contentDescription=""
+                    android:tint="?attr/colorOnSurface"
+                    app:srcCompat="@drawable/left_to_right_gradient" />
+
+                <TextView
+                    android:id="@+id/customiseCardPeriodLabel"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/input_spacing_large"
+                    android:text="Periodicity for the shown count:"
+                    android:textAppearance="@style/TextAppearance.Body" />
+
+                <androidx.appcompat.widget.AppCompatSpinner
+                    android:id="@+id/featureShowCountPeriodSpinner"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/dialog_input_spacing"
+                    android:entries="@array/feature_show_count_periods"
+                    android:spinnerMode="dropdown" />
+
+                <TextView
+                    android:id="@+id/customiseCardMethodLabel"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/input_spacing_large"
+                    android:text="Calculation method for the shown count:"
+                    android:textAppearance="@style/TextAppearance.Body" />
+
+                <androidx.appcompat.widget.AppCompatSpinner
+                    android:id="@+id/featureShowCountMethodSpinner"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/dialog_input_spacing"
+                    android:entries="@array/feature_show_count_methods"
+                    android:spinnerMode="dropdown" />
+
+            </LinearLayout>
+
+        </ScrollView>
+
+        <Button
+            android:id="@+id/closeButton"
```

Comment:
Can we please change this to a "DONE" button, use color accent and move it to the right of the dialog.